test/scripts/get-missing-cherry-picks.test.ts

```diff
@@ -0,0 +1,160 @@
+import assert from 'assert';
+import nock from 'nock';
```

Comment:
Sinon isn't a test runner, it's a testing util 😅
I kinda want to avoid adding Nock in the first place - do you suspect it's going to take a long time to make this switch in this PR?

Reply:
mocked using ts-mockito